pallets/staking-rewards/src/mock.rs

```diff
@@ -465,9 +465,41 @@ pub struct ExtBuilder {
     ideal_stake: Perquintill,
     target_inflation: Perquintill,
     filtered_accounts: Vec<AccountId>,
+    _phantom: PhantomData<T>,
+}
+
+impl<T> Default for ExtBuilder<T> {
+    fn default() -> Self {
+        Self {
+            initial_authorities: vec![],
+            stash: 0,
+            endowed_accounts: vec![],
+            endowment: 0,
+            root: None,
+            total_supply: 0,
+            non_stakeable: Default::default(),
+            pool_balance: 0,
+            ideal_stake: Default::default(),
+            target_inflation: Default::default(),
+            filtered_accounts: vec![],
+            _phantom: Default::default(),
+        }
+    }
 }
 
-impl ExtBuilder {
+impl<T> ExtBuilder<T>
+where
```

Comment:
Same as above: why generic over T and all these trait bounds? It worked just fine before, didn't it? Please enlighten me )